src/vs/workbench/contrib/terminal/browser/media/CodeTabExpansion.psm1

```diff
@@ -0,0 +1,57 @@
+# TODO: Dynamically enable depending on quality?
+Microsoft.PowerShell.Core\Register-ArgumentCompleter -CommandName "code","code-insiders" -Native -ScriptBlock {
+	param($wordToComplete, $commandAst, $cursorPosition)
```

Comment:
I noticed you're not using these parameters at all...

So are you just returning all the completions and relying on the VS Code side to filter them down or does PowerShell do some auto filtering?

If the former, do we need to go all the way down to PowerShell for this over somehow declaring this static completion in TypeScript only?

Reply:
This was a very quick version to add something to `code`, my plan is to build it out next iteration and figured what this gives now is better than nothing.